tests/unit/src/test/scala/tests/cli/CliSuite.scala

```diff
@@ -187,4 +189,24 @@ class CliSuite extends BaseCliSuite {
       |""".stripMargin
   )
 
+  checkCli(
+    "single-file",
+    """
+      |/index.md
+      |# Single file
+      |```scala mdoc
+      |println("one file")
+      |```
+      |""".stripMargin,
+    """
+      |/index.md
+      |# Single file
+      |```scala
+      |println("one file")
+      |// one file
+      |```
+      |""".stripMargin,
+    configureInFlag = { in => in + "/index.md" }
```

Comment:
To make this work on Windows
```suggestion
    configureInFlag = { in => in.resolve("index.md").toString }
```

Reply:
Ah, good call. Added this.